src/components/ModalVault.svelte

```diff
@@ -373,7 +373,7 @@
     <span>to insert a link</span>
   </div>
   <div class="prompt-instruction">
-    <span class="prompt-instruction-command">ctrl h</span>
+    <span class="prompt-instruction-command">ctrl g</span>
```

Comment:
ahh sorry I actually didn't mean to include that one yet -- I was thinking of making that an option as well.  The reason I changed it for myself is that ctrl-h is a common shortcut for backspace (a carry-over from emacs, but the key positions are easier to hit than backspace for me).

Reply:
I guess there's a question -- do you have any issue with changing it for everyone?  Or a preference on how I incorporate the customizable aspect then?  (a tickbox option to change it to ctrl-g, for example?). Let me know, and I'll make the appropriate changes.

Reply:
for now I've reverted it to ctrl-g, but would like to stay on your version, rather than having to have that one tweak on a version I manually install, so any feedback regarding integration would be appreciated :)

Cheers,
Kem

Reply:
> I guess there's a question -- do you have any issue with changing it for everyone?

_I_ don't have any issue with that, but I don't use it 🤷‍♂️ I just chose ctrl+h for "hide".

> a tickbox option to change it to ctrl-g, for example?

Hah, not really a fan of this 😅 Ultimately someone will ask for another keybind choice, or to change other shortcuts, and then we're cluttering the settings page with things that 99% of users don't care about.

Ideally all keybindings should be customizable (see #274) but I never got around to do it since it's a really low importance issue 🤷‍♂️